api/system.cc

```diff
@@ -158,6 +162,27 @@ void set_system(http_context& ctx, routes& r) {
             return json::json_return_type(json::json_void());
         });
     });
+
+    hs::dump_profile.set(r, [](std::unique_ptr<request> req) {
+        if (!__llvm_profile_dump) {
+            apilog.info("Executable is not instrumented with profile dumping, skipping");
```

Comment:
Skipping what?
Please mention in the log message what command was skipped. E.g.,
"dump_profile command skipped because Scylla executable was not built with coverage profiling". Or something.

Reply:
I will change it to "Profile will not be dumped, Executable is not instrumented with profile dumping"

Reply:
I have changed the text